functions/consumer/xmpp-consumer/src/test/java/org/springframework/cloud/fn/consumer/xmpp/XmppConsumerConfigurationTests.java

```diff
@@ -48,29 +47,23 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.awaitility.Awaitility.await;
-import static org.springframework.cloud.fn.test.support.xmpp.XmppTestContainerSupport.JOHN_USER;
-import static org.springframework.cloud.fn.test.support.xmpp.XmppTestContainerSupport.SERVICE_NAME;
-import static org.springframework.cloud.fn.test.support.xmpp.XmppTestContainerSupport.USER_PW;
 
 /**
  * @author Daniel Frey
  * @author Chris Bono
  */
-@SpringBootTest(
```

Comment:
Haha - you are un-polishing my polish (https://github.com/spring-cloud/stream-applications/commit/a9ed883124f27304be591de0d5b708e7193aa274#diff-7662de5b67155154da88bce749ea7b337d75a34724d1a8662a1e754c7a1d3f31R59). I prefer to have only the dynamic props in the DPS and make the static ones as SBT.props. 

I would rather you update the newly added https://github.com/spring-cloud/stream-applications/commit/985ab8b03578f28bd8378f4100348c340a6ac388#diff-94ab647cd1ad583f80e113271111c7a40988ea6490af95b74b9fb2e1dfa2108fR61 supplier tests to match this code as it was before this proposed change.